public/plugin.tsx

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I already made some of these changes in my custom expression PR to plugin.ts. Could you just make the necessary changes you need in `plugin.ts` without overwrite my code there. I think you accidently added new plugin.tsx file here instead of using current plugin.ts